example/tutorial/03_simple_view/simple_view.cpp

```diff
@@ -113,7 +113,8 @@ struct ReduceFunctor {
 };
 
 int main (int argc, char* argv[]) {
```

Comment:
Lets not change this example. This is not the main purpose of how ScopeGuard should be used.

Reply:
The idea was that using ScopeGuard would allow removing the braces that I put around the code. If we do change it, the commented-out initialize/finalize should be removed and the braces should be removed. Otherwise we can leave it with the braces.

Reply:
Restored the simple_view.cpp example to the pre-edit state ( includes initialize-finalize and braces, otherwise it fails to run).

Reply:
change pushed